Library/Formula/bibtex2html.rb

```diff
@@ -12,17 +10,34 @@ class Bibtex2html < Formula
     sha1 "32377bea1f584fedf5d2abb604a1d46e5e92ac5c" => :mountain_lion
   end
 
-  depends_on 'objective-caml'
-  depends_on 'hevea'
+  depends_on "objective-caml"
+  depends_on "hevea"
+  depends_on :tex => :optional
 
   def install
     # See: https://trac.macports.org/ticket/26724
-    inreplace 'Makefile.in' do |s|
-      s.remove_make_var! 'STRLIB'
+    inreplace "Makefile.in" do |s|
+      s.remove_make_var! "STRLIB"
     end
 
     system "./configure", "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.bib").write <<-EOS.undent
+      @article{Homebrew,
+          title   = {Something},
+          author  = {Someone},
+          journal = {Something},
+          volume  = {1},
+          number  = {2},
+          pages   = {3--4}
+      }
+    EOS
+    system "#{bin}/bib2bib", "test.bib", "--remove", "pages", "-ob", "out.bib"
+    assert File.read("test.bib") =~ /pages\s*=\s*{3--4}/
+    assert File.read("out.bib") !=~ /pages\s*=\s*{3--4}/
```

Comment:
Ok, it fails because `assert_not_match` returns false. It's kind weird. I bypass it by swapping the two `assert` order.